phpunit.xml.dist

```diff
@@ -7,6 +7,7 @@
          beStrictAboutChangesToGlobalState="true"
          colors="true"
          bootstrap="tests/bootstrap.php"
+         defaultTestSuite="MongoDB CRUD Test Suite"
```

Comment:
I don't believe the test suite was used previously. Perhaps it makes sense to rename this, since it's very much beyond CRUD. "Default Test Suite" would be fine.

Reply:
Done - I believe this was the original name, but it makes sense to finally change it.